lms/djangoapps/learner_dashboard/views.py

```diff
@@ -108,14 +108,11 @@ class ProgramDiscussionIframeView(APIView, ProgramSpecificViewMixin):
     def get(self, request, program_uuid):
         """ GET handler """
         program_discussion_lti = ProgramDiscussionLTI(program_uuid, request)
-        return Response(
-
-            {
-                'enabled': masters_program_tab_view_is_enabled(),
-                'discussion': {
-                    'iframe': program_discussion_lti.render_iframe(),
-                    'configured': bool(program_discussion_lti.configuration),
-                }
-            },
-            status=status.HTTP_200_OK
-        )
+        response_data = {
+            'tab_view_enabled': masters_program_tab_view_is_enabled(),
```

Comment:
will this also require a change to the frontend?

Reply:
yes, it was conveyed to me.

Reply:
YES!